ubuntu1604/templates/csv/services_enabled.csv

```diff
@@ -3,3 +3,4 @@ cron,cron,
 ntp,ntp,
 rsyslog,rsyslog,
 syslogng,syslogng,
+systemd_timesyncd
```

Comment:
@AxelNennker I believe that this should be `servicename,packagename,daemonname` e.g. `systemd_timesyncd,systemd,`